### PR TITLE
Added access token note

### PR DESCRIPTION
**Pull request by merveenoyan**
cc @osanseviero @coyotte508 @OlivierDehaene

### CODE REVIEW

**Comment by osanseviero** [comment]
Have we confirmed this works?

**Comment by merveenoyan** [comment]
@coyotte508 I feel like people don't check that out :/

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ A chat interface using open source models, eg OpenAssistant. It is a SvelteKit a
 If you don't want to configure, setup, and launch your own Chat UI yourself, you can use this option as a fast deploy alternative.
 
 You can deploy your own customized Chat UI instance with any supported LLM of your choice with only a few clicks to Hugging Face Spaces thanks to the Chat UI Spaces Docker template. Get started [here](https://huggingface.co/new-space?template=huggingchat/chat-ui-template).
+If you'd like to deploy a model with gated access or a model in a private repository, you can simply provide `HUGGING_FACE_HUB_TOKEN` in repository secrets. You need to set its value to an access token you can get from [here](https://huggingface.co/settings/tokens).
```
Comment (julien-c) [inline README.md:31]:
```suggestion
If you'd like to deploy a model with gated access or a model in a private repository, you can simply provide `HUGGING_FACE_HUB_TOKEN` in Space secrets. You need to set its value to an access token you can get from [here](https://huggingface.co/settings/tokens).
```

Comment (osanseviero) [inline README.md:31]:
```suggestion
If you'd like to deploy a model with gated access or a model in a private repository, you can simply provide `HUGGING_FACE_HUB_TOKEN` in [repository secrets](https://huggingface.co/docs/hub/spaces-overview#managing-secrets-and-environment-variables). You need to set its value to an access token you can get from [here](https://huggingface.co/settings/tokens).
```

**Review by osanseviero — APPROVED** [review]
LGTM but let's wait for a :+1: from a repo maintainer

**Review by nsarrazin — APPROVED** [review]

**Pull request merged**